ui/login_form.go

```diff
@@ -13,12 +13,14 @@ import (
 type LoginForm struct {
 	*tview.Form
 	Token chan string
+	Error chan error
 }
 
 func NewLoginForm() *LoginForm {
 	lf := &LoginForm{
 		Form:  tview.NewForm(),
 		Token: make(chan string, 1),
+		Error: make(chan error, 0),
```

Comment:
Channels are unbuffered by default. `0` is redundant here.
```suggestion
		Error: make(chan error),
```